src/sass/_variables.scss

```diff
@@ -9,3 +9,5 @@ $primary: #d58e32;
 $text-color: #2a2a2a;
 
 $form-border-color: #292929;
+
+$btn-more-color: #2a2a2a;
```

Comment:
When you have multiple variables with the same color it may be hard to keep them consist. Better would be refer to existing variable - `$text-color` - by setting it in this way: `$btn-more-color: $text-color;`. When you change the value of `text-color`, `btn-more-color` will be also updated.